Gopkg.toml

```diff
@@ -26,3 +22,7 @@
 [[override]]
   name = "github.com/docker/distribution"
   revision = "edc3ab29cdff8694dd6feb85cfeb4b5f1b38ed9c"
+
+[[override]]
+  name = "github.com/moby/moby"
+  revision = "4a804016ab8b9fd55a45cff9687a1de12dee5eb7"
```

Comment:
Remove following warning when running `dep`:

```
Warning: the following project(s) have [[constraint]] stanzas in Gopkg.toml:

  ✗  github.com/moby/moby

However, these projects are not direct dependencies of the current project:
they are not imported in any .go files, nor are they in the 'required' list in
Gopkg.toml. Dep only applies [[constraint]] rules to direct dependencies, so
these rules will have no effect.

Either import/require packages from these projects so that they become direct
dependencies, or convert each [[constraint]] to an [[override]] to enforce rules
on these projects, if they happen to be transitive dependencies,

Warning: Gopkg.lock is out of sync with Gopkg.toml or the project's imports.
```